earth_enterprise/src/fusion/portableglobe/servers/fileunpacker/build_and_test.py

```diff
@@ -78,13 +79,24 @@ def BuildLibrary(os_dir, ignore_results):
 
   configure_c_compiler(os_dir)
 
+  specialDefs = ''
+  if os_dir == "Windows":
+    # The logic below fixes a method used by the swig c++ wrapper. Mingw python headers
+    # should detect and fix this but for some reason they aren't working with mingw64
+    pythonCLib = "libpython" + sys.version + ".a"
```

Comment:
Python's sys.version returns something like '2.7.15 (v2.7.15:ca079a3ea3, Apr 30 2018, 16:30:26) [MSC v.1500 64 bit (AMD64)]' which probably is not what was intended here.